.github/workflows/native-images-tests.yml

```diff
@@ -47,17 +47,19 @@ jobs:
         -Pbuild-native-image,metrics-prometheus -am \
         -pl :hono-service-auth,:hono-service-command-router,:hono-service-device-registry-mongodb,\
         :hono-adapter-mqtt,:hono-adapter-amqp,:hono-adapter-coap,:hono-adapter-http,:hono-adapter-lora,\
-        :hono-tests
+        :hono-tests,:hono-service-device-registry-jdbc
```

Comment:
would you mind putting this after `:hono-service-device-registry-mongodb`?

Reply:
Done.